index.html

```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
     </section>
     
 </div>
+    <div class ="worksection">
+        <section id = "worksectionheader">
+        <h1 id= "worksectionheading"> My Recent Works</h1>
+        <img id="sectiondivider" src ="image/Vector (10).png">
+        </section>
+        
+        <section class="workdescription" >
+            <div id ="project1">
+                <img id ="placeholder" src="image/Img Placeholder.png">
+                <h2 id="project1text">Multi-Post Stories</h2>
+                <p id="project1supporttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul id ="project1language">
+                    <li class ="languageitem">css</li>
+                    <li class ="languageitem">html</li>
+                    <li class ="languageitem" id ="bootstrap">bootstrap</li>
+                    <li class ="languageitem">Ruby</li>
+                </ul>
+                <button id="seeproject1">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
```

Comment:
- [ ] Your HTML Structure is not using the right HTML semantics. The section element is a generic standalone section of the document, so use it to group elements that are related together. To know more about section tag [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
Please see below the simple structure that I suggest you follow for the works section of the portfolio: 

```
<section class="works">
    <div>
      <h2>My recent works</h2>
      <hr>
    </div>
    <!-- align the projects cards of the div below with display:grid -->
    <div class="grid here"> 
      <div class="project-cards feature-card">
        <img src="" alt="">
        <h2>Multipost Stories</h2>
        <p>Lorem ipsum dolor sit amet consectetur.</p>
        <ul>
          <li>CSS</li>
          <li>HTML</li>
          <li>Bootstrap</li>
          <li>Ruby</li>
        </ul>
        <button type="button">See project</button>
      </div>
      
      <div class="project-cards"></div>
      <div class="project-cards"></div>
      <div class="project-cards"></div>
      <div class="project-cards"></div>
      <div class="project-cards"></div>
    </div>
  </section>
```